gum/gumstalker.h

```diff
@@ -110,6 +110,122 @@ GUM_API void gum_stalker_unfollow (GumStalker * self, GumThreadId thread_id);
 GUM_API void gum_stalker_activate (GumStalker * self, gconstpointer target);
 GUM_API void gum_stalker_deactivate (GumStalker * self);
 
+/**
+ * This API is intended for use during fuzzing scenarios such as AFL forkserver.
+ * It allows for the child to feed back the addresses of instrumented blocks to
+ * the parent so that the next time a child is forked from the parent, it will
+ * already inherit the instrumented block rather than having to re-instrument
+ * every basic block again from scratch.
+ *
+ * This API has the following caveats:
+ *
+ * 1. This API is not intended to be used from JavaScript. It is unlikely there
+ *    will be scenarios where this is useful.
```

Comment:
I don't think we want to mention decisions made in language-bindings in Gum, as we're one layer below here and there should be a clear separation of concerns.